### PR TITLE
Showing

### PR DESCRIPTION
**Pull request by AnderskdKnudsen**

### CODE REVIEW

File: src/main/java/com/outofbounds/empire/Movies/Models/Movie.java [hunk 1]
```diff
@@ -1,13 +1,16 @@
 package com.outofbounds.empire.Movies.Models;
 
+import com.outofbounds.empire.Showings.Models.Showing;
+
 import javax.persistence.*;
+import java.util.List;
 
 @Entity
 @Table(name = "movies")
 public class Movie {
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Movies/Models/Movie.java]: remove spacing

File: src/main/java/com/outofbounds/empire/Movies/Models/Movie.java [hunk 2]
```diff
@@ -35,12 +41,23 @@ public Movie(String title, String description, int ageLimit, String category, fl
         this.ageLimit = ageLimit;
         this.category = category;
         this.price = price;
+//        this.showings = showings;
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Movies/Models/Movie.java]: remove comment

File: src/main/java/com/outofbounds/empire/Movies/Models/Movie.java [hunk 3]
```diff
@@ -35,12 +41,23 @@ public Movie(String title, String description, int ageLimit, String category, fl
...
         this.imageUrl = imageUrl;
     }
 
     // Non arg
     public Movie() {}
 
+    public List<Showing> getShowings()
+    {
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Movies/Models/Movie.java]: change to current line curly

File: src/main/java/com/outofbounds/empire/Movies/Models/Movie.java [hunk 4]
```diff
@@ -35,12 +41,23 @@ public Movie(String title, String description, int ageLimit, String category, fl
...
+        return showings;
+    }
+
+    public void setShowings(List<Showing> showings)
+    {
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Movies/Models/Movie.java]: change to current line curly

File: src/main/java/com/outofbounds/empire/Showings/Controllers/ShowingController.java [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * Created by Anders on 26-Sep-17.
+ */
+@RestController
+public class ShowingController {
+
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Showings/Controllers/ShowingController.java]: remove spacing

File: src/main/java/com/outofbounds/empire/Showings/Models/Showing.java [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+import java.util.Date;
+
+
+/**
+ * Created by Anders on 26-Sep-17.
+ */
+@Entity
+@Table(name = "showings")
+public class Showing
+{
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Showings/Models/Showing.java]: change to current line curly

File: src/main/java/com/outofbounds/empire/Showings/Models/Showing.java [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Showings/Models/Showing.java]: remove spacing

File: src/main/java/com/outofbounds/empire/Showings/Models/Showing.java [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+    @ManyToOne()
+    @JoinColumn(name = "showroom.id", nullable = false)
+    private Showroom showroom;
+
+    @Column(name = "datetime")
+    private Date datetime;
+
+    public Showing(Movie movie, Showroom showroom, Date datetime)
+    {
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Showings/Models/Showing.java]: change to current line curly

File: src/main/java/com/outofbounds/empire/Showings/Models/Showing.java [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+        this.movie = movie;
+        this.showroom = showroom;
+        this.datetime = datetime;
+    }
+
+    public Showing() {}
+
+    public int getId()
+    {
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Showings/Models/Showing.java]: change to current line curly

File: src/main/java/com/outofbounds/empire/Showings/Models/Showing.java [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+        return id;
+    }
+
+    public void setId(int id)
+    {
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Showings/Models/Showing.java]: change to current line curly

File: src/main/java/com/outofbounds/empire/Showings/Models/Showing.java [hunk 11]
```diff
@@ -0,0 +1,77 @@
...
+        this.id = id;
+    }
+
+    public Movie getMovie() {return movie;}
+
+    public void setMovie(Movie movie)
+    {
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Showings/Models/Showing.java]: change to current line curly

File: src/main/java/com/outofbounds/empire/Showings/Models/Showing.java [hunk 12]
```diff
@@ -0,0 +1,77 @@
...
+        this.movie = movie;
+    }
+
+    public Date getDatetime() {
+        return datetime;}
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Showings/Models/Showing.java]: move end-curly to next line

File: src/main/java/com/outofbounds/empire/Showings/Models/Showing.java [hunk 13]
```diff
@@ -0,0 +1,77 @@
...
+
+    public void setDatetime(Date datetime)
+    {
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Showings/Models/Showing.java]: change to current line curly

File: src/main/java/com/outofbounds/empire/Showings/Models/Showing.java [hunk 14]
```diff
@@ -0,0 +1,77 @@
...
+        this.datetime = datetime;
+    }
+
+    public Showroom getShowroom()
+    {
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Showings/Models/Showing.java]: change to current line curly

File: src/main/java/com/outofbounds/empire/Showings/Models/Showing.java [hunk 15]
```diff
@@ -0,0 +1,77 @@
...
+        return showroom;
+    }
+
+    public void setShowroom(Showroom showroom)
+    {
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Showings/Models/Showing.java]: change to current line curly

File: src/main/java/com/outofbounds/empire/Showings/Repositories/ShowingRepository.java [hunk 16]
```diff
@@ -0,0 +1,21 @@
...
+
+import java.util.List;
+
+/**
+ * Created by Anders on 26-Sep-17.
+ */
+@Transactional
+@Repository
+public interface ShowingRepository extends CrudRepository<Showing, Integer>
+{
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Showings/Repositories/ShowingRepository.java]: change to current line curly

File: src/main/java/com/outofbounds/empire/Showings/Repositories/ShowingRepository.java [hunk 17]
```diff
@@ -0,0 +1,21 @@
...
+
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Showings/Repositories/ShowingRepository.java]: remove spacing

File: src/main/java/com/outofbounds/empire/Showrooms/Models/Showroom.java [hunk 18]
```diff
@@ -1,18 +1,25 @@
...
 
+import com.outofbounds.empire.Showings.Models.Showing;
+
 import javax.persistence.*;
+import java.util.List;
 
 @Entity
 @Table(name = "showrooms")
 public class Showroom {
+
```
Comment (AnderskdKnudsen) [inline src/main/java/com/outofbounds/empire/Showrooms/Models/Showroom.java]: remove spacing

**Review by jasonkellydk — APPROVED** [review]
Super good

**Pull request merged**